Screenbox/Strings/en-US/Resources.resw

```diff
@@ -534,4 +534,70 @@
   <data name="FailedToLoadMediaNotificationTitle" xml:space="preserve">
     <value>Unable to load media</value>
   </data>
+  <data name="ThirdGenerationFileFriendlyName" xml:space="preserve">
+    <value>3GPP Multimedia File</value>
+  </data>
+  <data name="ThirdGeneration2FileFriendlyName" xml:space="preserve">
+    <value>3GPP2 Multimedia File</value>
+  </data>
+  <data name="AdvancedAudioFileFriendlyName" xml:space="preserve">
+    <value>Advanced Audio Coding File</value>
+  </data>
+  <data name="FlacFileFriendlyName" xml:space="preserve">
+    <value>Free Lossless Audio Codec File</value>
+  </data>
+  <data name="FlashFileFriendlyName" xml:space="preserve">
+    <value>Flash Video</value>
+  </data>
+  <data name="iTunesVideoFileFriendlyName" xml:space="preserve">
+    <value>iTunes Video</value>
+  </data>
+  <data name="MatroskaAudioFileFriendlyName" xml:space="preserve">
+    <value>Matroska Audio</value>
+  </data>
+  <data name="MatroskaVideoFileFriendlyName" xml:space="preserve">
+    <value>Matroska Video</value>
+  </data>
+  <data name="MicrosoftVideoFileFriendlyName" xml:space="preserve">
+    <value>Audio Video Interleave File</value>
+  </data>
+  <data name="MidiFileFriendlyName" xml:space="preserve">
+    <value>MIDI Music Data</value>
+  </data>
+  <data name="MpegFileFriendlyName" xml:space="preserve">
+    <value>MPEG Video</value>
+  </data>
+  <data name="MP3FileFriendlyName" xml:space="preserve">
+    <value>MP3 Audio</value>
+  </data>
+  <data name="MP4AudioFileFriendlyName" xml:space="preserve">
+    <value>MPEG-4 Audio</value>
+  </data>
+  <data name="MP4VideoFileFriendlyName" xml:space="preserve">
+    <value>MPEG-4 Video</value>
+  </data>
+  <data name="OggAudioFileFriendlyName" xml:space="preserve">
+    <value>Ogg Vorbis Audio</value>
+  </data>
+  <data name="OggVideoFileFriendlyName" xml:space="preserve">
+    <value>Ogg Video</value>
+  </data>
+  <data name="QuickTimeFileFriendlyName" xml:space="preserve">
+    <value>QuickTime Movie</value>
+  </data>
+  <data name="WaveFileFriendlyName" xml:space="preserve">
+    <value>WAVE Audio</value>
+  </data>
+  <data name="WebAudioFileFriendlyName" xml:space="preserve">
+    <value>WebM Audio</value>
+  </data>
+  <data name="WebMediaFileFriendlyName" xml:space="preserve">
+    <value>WebM Video</value>
+  </data>
+  <data name="WindowsMediaAudioFileFriendlyName" xml:space="preserve">
+    <value>Windows Media Audio</value>
+  </data>
+  <data name="WindowsMediaVideoFileFriendlyName" xml:space="preserve">
+    <value>Windows Media Video</value>
+  </data>
```

Comment:
I'm not sure if these names are widely recognizable. For example, "Audio Video Interleave" doesn't strike me as AVI at glance. By default, Windows uses "<extension> File" as the display name. If we override this, we will have to commit to localizing all display names in the future. Especially when many of these names are foreign to most people.

Reply:
> Nice work! There are some minor issues that I noticed when testing this. Personally, it feels a bit odd to have a custom icon shape for videos but not for audio. But I don't have any other idea right now and it still looks nice.

The audio it is custom (cd/vinyl album), I used the logo shapes just so it wasn't a big black blob,

>Can you also share your design file so new file types can be added in the future?

Sure I'll share the svg here. Sorry, had to remove the shadows because I was having problems.
![FileAssociationAudio](https://user-images.githubusercontent.com/698155/230736192-15dc704b-4f3e-4bef-99be-2d8babfa50a7.svg)
![FileAssociationVideo](https://user-images.githubusercontent.com/698155/230736193-1edaeea9-c1fe-4bf3-bfa6-8ed542d61f6a.svg)

Reply:
> I'm not sure if these names are widely recognizable. For example, "Audio Video Interleave" doesn't strike me as AVI at glance. By default, Windows uses `<extension> File` as the display name (e.g. "FLAC File", "AVI File"). If we override this, we will have to commit to localizing all display names in the future. Not sure if it's worth the effort, especially when many of these names are foreign to most people.

Some of them can be shortened, like the AVI and FLAC, but before (in the Windows Media Player times) used "proper" extension files. AVI was 'Video Clip', 3GPP was '3GPP Audio/Video' for example, now in the new Media Player (and Groove before) they do basically nothing and Windows just uses '<extension> File'